installing/installing_helm/install-helm-customization.adoc

```diff
@@ -51,16 +50,78 @@ $ helm install -n stackrox --create-namespace \
 ----
 <1> Use the `-f` option to specify the paths for your YAML configuration files.
 
+[id=generate-inti-bundle]
+== Generating an init bundle
+
+To create a secured cluster, you must create an init bundle. The secured cluster uses this bundle to authenticate with Central.
+
+You can create an init bundle using the the roxctl CLI or from the RHACS portal.
+
+[.discreet]
+=== Generating an init bundle by using roxctl CLI
+
 [id="install-roxctl-cli"]
-== Installing the roxctl CLI
+==== Installing the roxctl CLI
 `roxctl` is a command-line interface (CLI) for running commands on {product-title}.
 You can install the `roxctl` CLI to interact with {product-title} from a
 command-line interface. You can install `roxctl` on Linux, Windows, or macOS.
 
 // Installing the CLI by downloading the binary
-include::modules/install-roxctl-cli-linux.adoc[leveloffset=+2]
-include::modules/install-roxctl-cli-macos.adoc[leveloffset=+2]
-include::modules/install-roxctl-cli-windows.adoc[leveloffset=+2]
+include::modules/install-roxctl-cli-linux.adoc[leveloffset=+4]
+include::modules/install-roxctl-cli-macos.adoc[leveloffset=+4]
+include::modules/install-roxctl-cli-windows.adoc[leveloffset=+4]
 
 // Authenticating using CLI
-include::modules/cli-authentication.adoc[leveloffset=+2]
+include::modules/cli-authentication.adoc[leveloffset=+4]
+
+// Generating init bundle
+include::modules/roxctl-generate-init-bundle.adoc[leveloffset=+3]
+
+// Generating init bundle
+include::modules/portal-generate-init-bundle.adoc[leveloffset=+2]
+
+[id="configure-secured-cluster-services-helm-chart"]
+== Configuring the secured-cluster-services Helm chart
+
+This section describes Helm charts configuration parameters that you can use with the `helm install` and `helm upgrade` commands.
+You can specify these parameters by using the `--set` option or by creating YAML configuration files.
+
+Create the following files for configuring the Helm chart for installing {product-title}:
+
+* Public configuration file `values-public.yaml`: Use this file to save all non-sensitive configuration options.
+* Private configuration file `values-private.yaml`: Use this file to save all sensitive configuration options. Make sure that you store this file securely.
+//TODO: Add links to sample configuration files.
+
+[IMPORTANT]
+====
+While using the `secured-cluster-services` Helm chart, do not modify the `values.yaml` file that is part of the chart.
+====
+
+include::modules/secured-cluster-services-config.adoc[leveloffset=+2]
+
+[id="install-secured-cluster-services-helm-chart"]
+== Installing the secured-cluster-services Helm chart
```

Comment:
This should probably be its own module, with a `.Procedure` heading and everything. (Single step procedures use an unnumbered bullet).